scripts/regenerate.sh

```diff
@@ -99,6 +99,14 @@ Mgrpc/testing/test.proto=google.golang.org/grpc/interop/grpc_testing,\
 Mgrpc/testing/payloads.proto=google.golang.org/grpc/interop/grpc_testing,\
 Mgrpc/testing/empty.proto=google.golang.org/grpc/interop/grpc_testing
 
+# test feature use separate service package and only test one proto file
+protoc --go-grpc_out="${OPTS},use_separate_service_package=true:${WORKDIR}/out" \
+    "examples/route_guide/routeguide/route_guide.proto"
+# move service package to right place.
+mkdir -p "${WORKDIR}/out/google.golang.org/grpc/examples/route_guide/routeguide_grpc"
+mv "${WORKDIR}"/out/google.golang.org/grpc/examples/route_guide/routeguide/*_grpc.pb.go \
+  "${WORKDIR}/out/google.golang.org/grpc/examples/route_guide/routeguide_grpc"
```

Comment:
By convention, the directory name and the package names should be the same, see https://go.dev/blog/package-names#package-paths. Here, the package name can't be changed, so we should place the package under a directory named `routeguide`, e.g: `${WORKDIR}/out/google.golang.org/grpc/examples/route_guide/grpc/routeguide`

Reply:
I removed the goldenfile and used grep to check.